keps/prod-readiness/sig-auth/3225.yaml

```
@@ -0,0 +1,3 @@
+kep-number: 3225
```

This file has the wrong name too.

```suggestion
kep-number: 3325
```